src/checks/json-schema-errors.ts

```diff
@@ -95,5 +97,6 @@ export const jsonSchemaErrors: AslChecker = (definition, options) => {
                 instancePath: error.instancePath,
                 schemaPath: decodeURIComponent(error.schemaPath)
             }
-        }));
+        }))
+        .filter((v: StateMachineError,i,a)=>a.findIndex(v2=>(v2.schemaError.schemaPath===v.schemaError?.schemaPath && v2.schemaError.instancePath===v.schemaError?.instancePath))===i);
 }
```

Comment:
unrelated, but I noticed some duplicate error messages in an existing test after this change so this will remove any dupes before returning the errors